### ISSUE TITLE
Swap to Noun Project Sage icon for favicon?

### ISSUE THREAD

**Issue by eldain** [julietnpn/sage#26]

**Comment by julietnpn** [comment]
I have the sage icon now. What format, size, and color do we need it in?
https://thenounproject.com/search/?q=sage&i=1576732

**Comment by eldain** [comment]
@julietnpn Does The Noun Project allow you to download a simple SVG instance of the icon?
Could then run it through [https://realfavicongenerator.net/](https://realfavicongenerator.net/), though we're only using a single favicon.ico icon for this project.

**Comment by julietnpn** [comment]
Yep! I put it [here](https://github.com/julietnpn/sage/blob/master/frontend/static/frontend/img/noun_sage_1576732.svg)... though it should not be there permanently.
